src/routes/layout.css

```
@@ -1,2 +1,235 @@
+	--coral: #ff6b4a;
+	--lemon: #f4d35e;
+	--paper: #ffffff;
+	--muted: #66708a;
+	--line: #cbd7ee;
+	--line-strong: #9eb0d2;
+	--success: #18875d;
+	--danger: #c83939;
+	--font-display: 'Arial Rounded MT Bold', 'Trebuchet MS', ui-rounded, sans-serif;
+	--font-body: 'Avenir Next', Avenir, 'Segoe UI', system-ui, sans-serif;
+	--font-mono: 'SFMono-Regular', Consolas, 'Liberation Mono', monospace;
+	font-family: var(--font-body);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)